DOC-2004: Added docs for init_content_sync option (#2758)
tinymce/tinymce-docs: modules/ROOT/partials/configuration/init_content_sync.adoc

[[init_content_sync]]
== `+init_content_sync+`

> This option enables you to change the way the iframe is initialized from using srcdoc and being async to using document.write and being sync. Setting
> this to `true` will reduce some initialization flicker. But some flicker like loading the content css in could still occur.

Reviewer: ```suggestion
This option determines whether the iframe is initialized synchronously or asynchronously.

When disabled, the editor is initialized asynchronously using `srcdoc`, however this can occasionally cause the editor to flicker during initialization.
When enabled, the editor will be initialized using `document.write`, which can reduce this flickering, but some flicker could still occur, such as loading the content css.
```
Mainly wanted to move the word "Setting" onto the correct line, but thought I'd attempt a rewrite.

Author: That is so much better. Thanks a ton.

[[init_content_sync]]
== `+init_content_sync+`

This option determines whether the iframe is initialized synchronously or asynchronously.

> When disabled, the editor is initialized asynchronously using `srcdoc`, however this can occasionally cause the editor to flicker during initialization.
> 
> When enabled, the editor will be initialized using `document.write`, which can reduce this flickering, but some flicker could still occur, such as loading the content css.

Reviewer 2: ```suggestion
When disabled, the editor is initialized asynchronously using `srcdoc`. This can, occasionally, cause the editor to flicker during initialization.

When enabled, the editor is initialized synchronously, using `document.write`. This can reduce flickering.

Flickering can still occur during synchronous initialization, however, such as when loading the content css.
```

Re-structured to go all in on parallelism.

Paragraphs 1 and 2 have completely parallel structure:

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_State; action, source. Consequence._

Not to argue the grand literary virtue of the above sentences, but parallelism is enormously helpful in getting people to remember what they’ve read or heard. (Perhaps the most famous example is Neil Armstrong’s statement on stepping onto the Moon: _That’s one small step for a man; one giant leap for mankind_.)

In this structure, the 3rd paragraph functions as a postscript: a supplement that adds a specific detail to the information in the parallel structure. A detail that will also be remembered more readily because said detail connects directly to the easily-remembered parallel structure.
